visualstimulation/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .utils import *
```

Comment:
I believe it is best to avoid import "all". The chance of future changes breaking everything is higher with import*